internal/pkg/schema/schema.go

```diff
@@ -72,6 +72,12 @@ func validateJsonSchema(schema map[string]interface{}, content *orderedmap.Order
 		parametersMap = utils.NewOrderedMap()
 	}
 
+	// Skip empty configurations.
+	// Users often just create configuration in UI, but leaves it unconfigured.
+	if len(parametersMap.Keys()) == 0 {
```

Comment:
Ak konfiguracia config/row neobsahuje kluc `parameters` alebo je JSON objekt prazdny -> tak sa pri validacii preskoci.

Uzivatelia casto vytvoria v UI konfiguraicu, ale uz ju nenastavia.
Pokom chcu pouzit CLI, a padam im to na prazdnych konfiguraciach.